CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,50 @@
+
+Blockchain Governance Initiative Network (“BGIN” or “we”) sets forth the following Code of Conduct (the "CoC") to help guide discussion in forums and meetings regardless of their form, including but not limited to, any in person meetings, online discussions, mailing lists, and video conferences (each a “Meeting,” and collectively, the “Meetings”). For all participants, we hope that complying with the CoC will make the Meetings safe, respectful, and fruitful for all participants.
+
+1. The goal of all Meetings is to serve as a neutral forum to discuss and, if necessary debate, how blockchain-based networks should be governed and how blockchain technology can be used for society. Participants are required to read through the Core Values of BGIN [^1] before joining the meetings. Participants should refrain from advocating for the use of specific technologies, services in principle. In particular, excessive advertisements for a particular product or service will be prohibited at the Meetings. To maintain the neutrality of BGIN, all the participants agree to participate in any BGIN activities on an individual basis and not as representatives of the organizations they belong to or are affiliated with.
+
+2. The scope of application of the CoC shall be as follows:
+
+   (a) The venue where the meetings are held;
+
+   (b) Remarks during networking activities inside and outside the venue, such as, including but not limited, to welcome reception and dinners; and
+
+   (c) Online statements related to the Meetings
+
+3. In principle, the intellectual property rights of the presenters at the Meetings belong to the participants.
+
+4. We prohibit harassment, unfair sabotage, or aggressive behavior as listed below at any Event, including welcome reception and dinners;
+
+   (a) harassment based on the participant’s race, gender, age, sexual orientation, disability, physical appearance, nationality, ethnicity, religion, affiliation, or attributes;
+
+   (b) offensive comments;
+
+   (c) verbal threats or demands;
+
+   (d) sexual expressions in public places;
+
+   (e) threatening behavior;
+
+   (f) stalking;
+
+   (g) sabotage of sessions;
+
+   (h) undesirable contact or observations by the other party
+
+   (i) any other act of coercing others which is not suitable for the Meetings
+
+5. All conversations at the Meetings shall be convened in accordance with Chatham House Rule [^2]
+
+6. Any participant who is requested by Acting Co-Chairs to stop harassment stated in paragraph 4 must immediately comply with the instructions of the Organizers. The participant who violates the CoC may be expelled from the venue at the discretion of the Organizers and where applicable they do so without any refunding fee paid by the participant (if applicable).
+
+7. Any concerns or questions concerning the CoC should be sent to ​bgin-contact@bgin.team​.
+
+[^1] https://github.com/bgin-global/genesis-documents/blob/master/Genesis.md#our-core-values
```

Comment:
@schwentker Thank you. 

One change request. I don't think [^1] notation works at GitHub. can you add a commit to remove `^`  ?